src/Conference/AddConferenceForm.jsx

```diff
@@ -1,10 +1,12 @@
 import React, { useState } from "react";
-import Input from "./Input";
-import { createNewConference } from "./api";
 import { useHistory } from "react-router-dom";
+import { withCookies } from "react-cookie";
+import Input from "../Input";
+import { createNewConference } from "../api";
 
-const Form = () => {
+export const AddConferenceForm = ({ allCookies = {} }) => {
   let history = useHistory();
+  const token = allCookies.sessionToken.token;
```

Comment:
if `allCookies` is equal to your default, this will break as you can read property token from undefined. 
`allCookies.sessionToken ? allCookies.sessionToken : null` is one work around

Reply:
OK just replaced it with `const token = allCookies.sessionToken ? allCookies.sessionToken.token : null` - it's still working so hope that's alright